zapisy/apps/enrollment/courses/templates/courses/course_parts/groups_section.html

```diff
@@ -131,4 +131,4 @@ <h3 class="d-inline-block">{{ class_type|decode_class_type_plural }}
         <p><strong>UWAGA!</strong>
             Wyższa liczba oznacza wyższy priorytet, po zapisaniu do grupy zostajemy usunięci z kolejek o niższym priorytecie.
         </p>
-    </div>
\ No newline at end of file
+    </div>
```

Comment:
Jedynie na potrzeby tego pliku korzystamy z `zapisy/apps/common/assets/main-end/expose_libs.ts`
A jednak plik jest wywoływany dla każdej podstrony. Proponuję przenieść wywołanie do tego pliku.
```suggestion
    </div>
    {% render_bundle "courses-course-list-group" %}
```

Reply:
Rezygnujemy z pliku `zapisy/apps/common/assets/main-end/expose_libs.ts`, komentarz nieaktualny